pkgs/petsc/petsc.py

```diff
@@ -108,9 +108,14 @@ def configure(ctx, stage_args):
         conf_lines.append('--with-suitesparse-include=${SUITESPARSE_DIR}/include/suitesparse')
         conf_lines.append('--with-suitesparse-lib=[${SUITESPARSE_DIR}/lib/libumfpack.a,libklu.a,libcholmod.a,libbtf.a,libccolamd.a,libcolamd.a,libcamd.a,libamd.a,libsuitesparseconfig.a]')
 
+    if 'SUPERLU_DIST' in ctx.dependency_dir_vars:
+        conf_lines.append('--with-superlu_dist=1')
+        conf_lines.append('--with-superlu_dist-include=${SUPERLU_DIST_DIR}/include/superlu_dist')
+        conf_lines.append('--with-superlu_dist-lib=[${SUPERLU_DIST_DIR}/lib/libsuperlu_dist.a]')
```

Comment:
This looks dangerous.  Won't the `.a` library be non-PIC and therefore unusable in a shared library?